Our.Umbraco.TagHelpers/ImgTagHelper.cs

```diff
@@ -314,12 +314,63 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             // The configuration allows us to define whether images are configured "mobile first". This simply alternates between min-width & max-width media queries.
             var imageSizes = GetImageSizes(MediaItem != null);
 
+            // Avoid rendering a WebP alternative if the image is a GIF
+            var imageFormat = MediaItem != null ? MediaItem.Url().Split('.').Last() : FileSource?.Split('.').Last(); 
```

Comment:
Assume this is getting the file extension, there is `Path.GetExtension` that removes the need for a bespoke approach.